docs/manual/spec.md

```diff
@@ -472,6 +481,18 @@ software).
 
 ### Sub-sections
 
+#### `%sourcelist`
+
+List of sources, one per row. Handled like unnumbered Source tags. For
```

Comment:
I'd use "line" instead of "row" here, just to be consistent with the rest of the file.